### ISSUE TITLE
Fix NPM Errors

### ISSUE THREAD

**Issue by mbeckenbach** [nicky-lenaers/ngx-scroll-to#108]
Looks like the package runs fine on angular 7.
Could you please update your dependencies?

```
@nicky-lenaers/ngx-scroll-to@1.1.1" has incorrect peer dependency "@angular/core@^6.0.0".
```

**Comment by BergerLand** [comment]
I also ran into the same.

**Comment by rafa-as** [comment]
Same here!

**Comment by zahirkelloud** [comment]
Please fix your peer deps

**Comment by nicky-lenaers** [comment]
This is now fixed in [2.0.0](https://github.com/nicky-lenaers/ngx-scroll-to/releases/tag/2.0.0). Thanks for reporting!
